src/lamp_py/ingestion/convert_gtfs_rt.py

```diff
@@ -222,7 +222,7 @@ def process_files(self) -> Iterable[pyarrow.table]:
         process_logger.log_complete()
 
     def yield_check(
-        self, process_logger: ProcessLogger, min_rows: int = 5_000_000
+        self, process_logger: ProcessLogger, min_rows: int = 2_000_000
```

Comment:
Reduced yield size to allow for the smaller ECS instance size of the ad-hoc process. This appears to not have any significant impact on ingestion processing speed.